level1/f256/modules/vtio.asm

```diff
@@ -345,8 +365,29 @@ InitKeyboard        clr       D.KySns
 ex@                 ldd        #0 set D to 0
                     std       V.KeyDrvMPtr,u clear the module pointer
                     std       V.KeyDrvEPtr,u clear the entry pointer
+                    rts       return to the caller
+
+		    ifgt      Level-1
```

Comment:
Why is this conditional here if we support Level 1?

Reply:
Removed support for Level-1 to remove from FEU.  Since FEU uses L1 vtio, it was easier to strip the mouse from L1 than to make a conditional for the FEU.